spec/load_fixtures_helper.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper.rb'
+
+RSpec.configure do |rspec|
+  # This config option will be enabled by default on RSpec 4,
+  # but for reasons of backwards compatibility, you have to
+  # set it on RSpec 3.
+  #
+  # It causes the host group and examples to inherit metadata
+  # from the shared context.
+  rspec.shared_context_metadata_behavior = :apply_to_host_groups
+end
+
+RSpec.shared_context "fixture moabs in db" do
+  before(:all) do
+    setup
+    load_fixture_moabs
+  end
+  after(:all) do
+    # TODO: danger - if there are objects in the test db we want to keep
+    Settings.moab.storage_roots.each do |_name, storage_root|
+      storage_dir = File.join(storage_root, Settings.moab.storage_trunk)
+      PreservationCopy.where(endpoint_id: Endpoint.find_by(storage_location: storage_dir).id).each do |pc|
```

Comment:
prob not worth it in this PR, but as a quick tech debt thing ticket, we might want to centralize this `storage_dir = File.join(storage_root, Settings.moab.storage_trunk)` thing into some little helper method somewhere, since we seem to be using it in at least a couple places (happy to create a ticket if that seems like a good idea).

Reply:
great idea!  Please do make a ticket.

Reply:
filed https://github.com/sul-dlss/preservation_core_catalog/issues/171